tests/braille-specs/spaces.yaml

```diff
@@ -1606,33 +1606,6 @@ tests:
   - ['\x0020', '\x0020']
   - ['\x00a0', 'a']
 flags: {testmode: forward}
-tests:
-  - ['\x000a', '\x0020']
-  - ['\x000b', '\x0020']
-  - ['\x000c', '\x0020']
-  - ['\x000d', '\x0020']
-  - ['\x2000', '\x0020']
-  - ['\x2001', '\x0020']
-  - ['\x2002', '\x0020']
-  - ['\x2003', '\x0020']
-  - ['\x2004', '\x0020']
-  - ['\x2005', '\x0020']
-  - ['\x2006', '\x0020']
-  - ['\x2007', '\x0020']
-  - ['\x2008', '\x0020']
-  - ['\x2009', '\x0020']
-  - ['\x200a', '\x0020']
-  - ['\x2028', '\x0020']
-  - ['\x2029', '\x0020']
-  - ['\x202f', '\x0020']
-  - ['\x205f', '\x0020']
```

Comment:
Why are these tests removed? Aren't they for testing `or.tbl`?

Reply:
I think it merely appears that way in the github differences because the tests were exactly the same for or.tbl and pa.tbl. If you look at the whole file the tests are still there for or.tbl

Reply:
OK